src/components/dialogs/AddStaffNotationDialog.vue

```diff
@@ -230,7 +230,7 @@ export default class AddStaffNotationDialog extends Vue {
       // Administrative Dissolution, Put Back On, Freeze/Unfreeze Business require a detailed comment
       (v: string) => (
         !!v ||
-        (!this.isAdministrativeDissolution && !this.isPutBackOn & !this.isAdministerFreeze) ||
+        (!this.isAdministrativeDissolution && !this.isPutBackOn && !this.isAdministerFreeze) ||
```

Comment:
The missing `&` turned this result into a number, which the unit tests complained about. Possibly this all resolved to True or False anyway so it wasn't a functional bug.